.env.sample

```
@@ -1,3 +1,20 @@
+NUXT_CLOUDFLARE_NAMESPACE_ID=
+NUXT_CLOUDFLARE_API_TOKEN=
+
+## SUPABASE
+NUXT_DATABASE_NAME=
+NUXT_DATABASE_PASSWORD=
+
+# Connect to Supabase via connection pooling
+NUXT_DATABASE_URL=
+
+# Direct connection to the database. Used for migrations
+NUXT_DIRECT_URL=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)